nixos/modules/services/networking/zerotierone.nix

```diff
@@ -38,6 +49,9 @@ in
     # ZeroTier does not issue DHCP leases, but some strangers might...
     networking.dhcpcd.denyInterfaces = [ "zt0" ];
 
+    # ZeroTier receives UDP transmissions on port 9993 by default
+    networking.firewall.allowedUDPPorts = [ 9993 ];
```

Comment:
We should first discuss this change here: https://github.com/NixOS/nixpkgs/pull/37949/files#r192686531